tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.ts

```diff
@@ -48,90 +46,97 @@ export class LinkedTimeFobControllerComponent {
   @ViewChild('startFobWrapper') readonly startFobWrapper!: ElementRef;
   @ViewChild('endFobWrapper') readonly endFobWrapper!: ElementRef;
   @Input() axisDirection!: AxisDirection;
```

Comment:
Let's add a new line between "viewChild" group and "input" groups. See example
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/widgets/histogram/histogram_component.ts;l=90